package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "4.4.9",
+  "version": "4.5.0",
```

Comment:
Yes, this is a significant new feature, so I agree with increasing the minor release number.